Backpack-SwiftUI/Snippet/Classes/BPKSnippet.swift

```diff
@@ -0,0 +1,104 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2022 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKSnippet: View {
+    let image: Image
+    let accessibilityLabel: String
+    let headline: String?
+    let description: String?
+    let bodyText: String?
+    let imageOrientation: ImageOrientation
+    let onClick: (() -> Void)?
+    
+    public init(
+        image: Image,
+        accessibilityLabel: String,
+        headline: String? = nil,
+        description: String? = nil,
+        bodyText: String? = nil,
+        imageOrientation: ImageOrientation = .landscape,
+        onClick: (() -> Void)? = nil
```

Comment:
Totally not on you/your PR, but we need to find a convention on how. we are calling these actions. Click or tap? I see mixed usages across backpack

Reply:
That's correct, the `portrait` mode is tall enough to not show the bottom in an iPhone SE size:

https://github.com/Skyscanner/backpack-ios/assets/26660989/cf0f1f74-d151-443a-a291-8d9da8a174a0


> Totally not on you/your PR, but we need to find a convention on how. we are calling these actions. Click or tap? I see mixed usages across backpack

Agreed, just followed API table as strictly as possible

Reply:
The table might have been agreed on with Web in mind. In browsers, both, mouse clicks and touch screen pointers are using same `click` event, hence the name.